zcash_client_backend/src/data_api/chain.rs

```diff
@@ -363,6 +364,17 @@ where
                                 )
                                 .unwrap()
```

Comment:
```suggestion
                                .expect("the number of Sapling outputs in a block cannot overflow u32")
```

Reply:
Also, we have at least two places where the number of outputs/actions in a block is computed. Should it be a method returning `u32` (well, two methods: `output_count` and `action_count`)?